src/components/media.js

```diff
@@ -28,6 +34,15 @@
     const variable = imageSource && imageSource.findIndex(v => v.name) !== -1;
     const variableDev = env === 'dev' && (variable || !imgUrl);
 
+    const hasLink = linkTo && linkTo.id !== '';
```

Comment:
I would write the following
```
    const hasLink = linkTo && linkTo.id !== '';
    const hasExternalLink = linkToExternal && linkToExternal.id !== '';
    const anyLink = hasLink || hasExternalLink;
```
as this:
```
    const hasInternalLink = linkTo && linkTo.id !== '';
    const hasExternalLink = linkToExternal && linkToExternal.id !== '';
    const hasLink = hasInternalLink || hasExternalLink;
```
Variables with names starting with `is` and `has` mean they contain a boolean value. When it starts with `any`, it gives another idea. Remember that the current `anyLink` is used in the other code, so that has to be renamed as well.